mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -212,7 +229,8 @@ view: subscriptions__events {
 }
 
 view: subscriptions__retention {
-  dimension: retention_month {
+  dimension: age_in_months {
```

Comment:
this is used in both retention and churn, and calling it `retention month` on the churn dashboard was confusing.